payroll/models.py

```diff
@@ -229,17 +229,17 @@ class EmployerAlias(AliasModel):
     def clean(self):
         super().clean()
 
-        if self.employer.is_department:
-            duplicate_alias = type(self).objects.filter(
-                Q(name=self.name) & Q(employer__parent_id=self.employer.parent.id)
-            )
-        else:
-            duplicate_alias = type(self).objects.filter(
-                Q(name=self.name) & Q(employer__parent_id__isnull=True)
-            )
+        existing_alias = type(self).objects.filter(
```

Comment:
Refinement of existing logic. Aliases are always associated with the employer ID, so we can use the same logic for units and departments.